src/components/list-illustration/list-illustration-item.html.twig

```diff
@@ -96,5 +100,9 @@
   {% endif %}
   </div>
 </li>
-
+{% if divider and not last %}
+  <li class="ecl-list-illustration__divider">
```

Comment:
Couldn't we set a css border, instead of rendering a full `<li>`  for this?
The `last` could be used to add a css class to the last item, instead of relying only on twig

Reply:
well, yes, in the end i had to, but it wanted to use once this silly component we introduced.. :)